src/pcapi/routes/pro/venues.py

```diff
@@ -89,6 +90,30 @@ def edit_venue(venue_id: str, body: EditVenueBodyModel) -> GetVenueResponseModel
     venue_attrs = body.dict(exclude=not_venue_fields, exclude_unset=True)
     venue = offerers_api.update_venue(venue, body.contact, **venue_attrs)
 
+    accessibility_changes = {
+        field: body[field] != venue[field]
+        for field in [
+            "audioDisabilityCompliant",
+            "mentalDisabilityCompliant",
+            "motorDisabilityCompliant",
+            "visualDisabilityCompliant",
+        ]
+    }
+    if True in accessibility_changes and body.isAccessibilityAppliedOnAllOffers:
+        edit_accessibility = {
+            field: body[field] != venue[field]
+            for field in [
+                "audioDisabilityCompliant",
+                "mentalDisabilityCompliant",
+                "motorDisabilityCompliant",
+                "visualDisabilityCompliant",
+            ]
+        }
+        update_all_venue_offers_accessibility_job.delay(venue, edit_accessibility)
```

Comment:
J'ai eu un peu de mal à comprendre l'ensemble du fonctionnement et les différentes étapes. Je te propose quelque chose dans ce genre là, à voir si tu trouves ça plus lisible également (pour le nommage je n'ai pas mieux mais je sais que tu trouveras ;))

```suggestion
    accessibility_fields = ["audioDisabilityCompliant", "mentalDisabilityCompliant", "motorDisabilityCompliant", "visualDisabilityCompliant"]
    at_least_one_accessibility_field_updated = any([body[field] != venue[field] for field in accessibility_fields])

    if at_least_one_accessibility_field_updated and body.isAccessibilityAppliedOnAllOffers:
        edited_accessibility = {
            field: body[field]
            for field in accessibility_fields
        }
        update_all_venue_offers_accessibility_job.delay(venue, edited_accessibility)
```